cobra/cmd/root.go

```diff
@@ -73,7 +73,10 @@ func initConfig() {
 
 	viper.AutomaticEnv()
 
-	if err := viper.ReadInConfig(); err == nil {
-		fmt.Fprintln(os.Stderr, "Using config file:", viper.ConfigFileUsed())
+	if err := viper.ReadInConfig(); err != nil {
+		cobra.CheckErr(err)
```

Comment:
The config file is optional so we shouldn't exit on all errors.
You can check if the file was not found by checking for an `viper.ConfigFileNotFoundError` error.